### PR TITLE
Add groupValue filter to pageSearchModal

### PR DESCRIPTION
**Pull request by ellsclytn**

### CODE REVIEW

File: src/components/search/PageSearchModal/index.js [hunk 1]
```diff
@@ -84,6 +86,7 @@ module.exports = React.createClass({
       page: page || 1,
       pageSize: this.props.pageSize,
       pageType: this.props.pageType,
+      groupValue: this.props.groupValues
```
Comment (slwen) [inline src/components/search/PageSearchModal/index.js]: Should this key be named `groupValues` (plural)?

Comment (ellsclytn) [inline src/components/search/PageSearchModal/index.js]: No. Pages API expects it as `groupValue`, regardless of being an array or string.

Comment (bradparker) [inline src/components/search/PageSearchModal/index.js]: This is going to keep coming up. Can we add an issue to fix this after this PR has gone through?
I know @ellsclytn has to get it out quickly, and this was here before the work this PR concerns.
@slwen ?

Comment (slwen) [inline src/components/search/PageSearchModal/index.js]: Sure :+1:

**Pull request merged**